_data/toc.yaml

```diff
@@ -1392,11 +1392,14 @@ manuals:
       title: Hello Build
     - sectiontitle: Buildx
       section:
-        - path: /build/buildx/
-          title: Working with Buildx
         - path: /build/buildx/install/
           title: Install Buildx
-
+        - path: /build/buildx/
```

Comment:
Not sure about the order, looks confusing to me to have the parent page after a subpage.

As I can see with compose it's the first page: https://docs.docker.com/compose/ so we should be consistent.

Reply:
Maybe `Overview` and `Install` is enough as it's already a subsection of `Buildx`. Don't think we need to repeat ourselves?

Reply:
I see what you mean, that there's hierarchy providing context. But when we need to derive it from the folder context and there's multiple in the doc set, I think it's best for any "just" 'overview' or 'install' to be the ones for the main section (so doc set root index or install) for clear disambiguation in a glance.